myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -50,7 +50,7 @@ def _setup(self,
                goal_center=np.array([0.4, -0.25, 1.05]),
                max_force=1500,  # Max force against throwing
 
-               proximity_th=0.15,  # object-target proximity threshold
+               proximity_th=0.03,  # object-target proximity threshold
```

Comment:
Update this value to 0.03, the idea is to have 10cm in each dimensions. 0.1^2 *3 = 0.03. The pillar radius is 15cm and the box height from pillar is maximum 4cm. Thus 10cm in each direction would suffice in this case.

Reply:
I think we need to take the sqrt of it to get the Euclidean distance. So should be around 0.17?